MacPass/de.lproj/Localizable.strings

```diff
@@ -313,17 +313,17 @@
 "MESSAGE_ADD_PLUGIN_OPEN_PANEL" = "Bitte wählen Sie das Plugin aus, welches installiert werden soll";
 
 /* Message in the open panel to import an XML file */
-"MESSAGE_XML_OPEN_PANEL" = "Bitte wählen Sie die XML-Datei aus, die importiert werden soll.";
+"MESSAGE_XML_OPEN_PANEL" = "Bitte wählen Sie die .XML-Datei aus, die importiert werden soll.";
 
 /* Menu item to toggle display of modified date column in entry table
    Modification date column title */
 "MODIFIED" = "Verändert";
 
 /* Action title for moving an entry via drag and drop */
-"MOVE_ENTRY" = "Eintrage bewegen";
+"MOVE_ENTRY" = "Einträge bewegen";
```

Comment:
This is something that might change. There are pluralised versions as well so I would keep to the singular here.

Reply:
Since the PR author doesn't react I'll comment this. Einträge would be the correct pluralised version. Singular is "Eintrag", "Eintrage" is neither plural nor singular and should be changed to one of them.

Reply:
@nicoa your right. The typo needs to be fixed surely.